tests/ui_tests/app/page/common/tooltip_links/page.py

```diff
@@ -72,3 +72,22 @@ def check_warn_button(self, tab_name: str, expected_warn_text: list):
         assert (
             expected_warn_text == warn_text_popup
         ), f"Expected warning is {expected_warn_text}, but actual is {warn_text_popup}"
+
+    def check_toolbar_elements(self, tab_names: [str]):
+        self.assert_displayed_elements([CommonToolbarLocators.admin_link])
+        tab_names_upper = [tab_name.upper().strip("_") for tab_name in tab_names]
+
+        for tab_name in tab_names_upper:
+            self.assert_displayed_elements([CommonToolbarLocators.text_link(tab_name)])
+
+        tab_names_actual = (
+            self.find_element(CommonToolbarLocators.all_links)
+            .text.replace("\n", "")
+            .replace("play_circle_outline", "")
+            .replace("sync_problem", "")
+            .replace("priority_hight", "")
+        )
```

Comment:
Maybe add a little comment here? For us to remember why it's done this way

Reply:
added